Formula/r/rustcat.rb

```diff
@@ -0,0 +1,25 @@
+class Rustcat < Formula
+  desc "Modern Port listener and Reverse shell"
+  homepage "https://github.com/robiot/rustcat"
+  url "https://github.com/robiot/rustcat/archive/refs/tags/v3.0.0.tar.gz"
+  sha256 "59648e51ab41e4aeb825174dfbb53710207257feb3757521be98ed28c4249922"
+  license "GPL-3.0-only"
```

Comment:
Sorry, maybe I wasn't clear: `GPL-3.0` does not imply `GPL-3.0-only`. Can you ask upstram to confirm whether later GPL licenses apply to this software? Thanks!

Reply:
https://github.com/robiot/rustcat/issues/65

@chenrui333 has already inquired with the author, awaiting a reply.

Reply:
@7a6163 you can use GPL-3.0-only here. I will update the cargo.toml, but I can't publish a new version atm.

Reply:
Thanks!